bedrock/firefox/templates/firefox/installer-help.html

```diff
@@ -67,3 +67,7 @@ <h3 class="installer-help-emphasis-title">{{ _('Need help installing?') }}</h3>
 {% endblock %}
 
 {% block js %}{% endblock  %}
+
+{% block experiments %}
+  {{ js_bundle('installer_help_experiment') }}
```

Comment:
Why did you remove the switch here? I'd suggest we keep it, so we can turn the experiment on/off independent of a code review and production push?

Reply:
I had removed it when I had pushed this to a demo branch for design review & I forgot to put it back in. Adding back now.

Reply:
Ah, demos are treated like `Dev=True`, so switches are on by default when you push there. It should "just work" ™️ without needing to remove.